spk/gerrit/src/installer.sh

```diff
@@ -0,0 +1,133 @@
+#!/bin/sh
+
+# Package
+PACKAGE="gerrit"
+DNAME="Gerrit"
+
+# Others
+INSTALL_DIR="/usr/local/${PACKAGE}"
+SSS="/var/packages/${PACKAGE}/scripts/start-stop-status"
+JAVA_DIR="/var/packages/JavaManager/target/Java/jre"
+GIT_DIR="/usr/local/git"
+PATH="${INSTALL_DIR}/bin:${GIT_DIR}/bin:${JAVA_DIR}/bin:/usr/local/bin:/bin:/usr/bin:/usr/syno/bin"
+USER="gerrit"
+GROUP="users"
+GIT="${GIT_DIR}/bin/git"
+
+SERVICETOOL="/usr/syno/bin/servicetool"
+FWPORTS="/var/packages/${PACKAGE}/scripts/${PACKAGE}.sc"
+
+BOUNCYCASTLE_VER="151"
+BOUNCYCASTLE_PROVIDER_URL="http://www.bouncycastle.org/download/bcprov-jdk15on-${BOUNCYCASTLE_VER}.jar"
+BOUNCYCASTLE_SSL_URL="http://www.bouncycastle.org/download/bcpkix-jdk15on-${BOUNCYCASTLE_VER}.jar"
+
+#SYNO_GROUP
+#SYNO_GROUP_DESC
+
+DEFAULT_PLUGINS="download-commands review"
+
+preinst ()
+{
+	exit 0
+}
+
+postinst ()
+{
+	# Link
+	ln -s ${SYNOPKG_PKGDEST} ${INSTALL_DIR}
+
+	# Install busybox stuff
+	${INSTALL_DIR}/bin/busybox --install ${INSTALL_DIR}/bin
+
+	if [ "${SYNOPKG_PKG_STATUS}" != "UPGRADE" ]; then
+		# Create user
+		adduser -h ${INSTALL_DIR} -g "${DNAME} User" -G ${GROUP} -s /bin/sh -S -D ${USER}
+
+		# Install prerequsite libraries
+		wget -nv -c "$BOUNCYCASTLE_PROVIDER_URL" -O "${SYNOPKG_PKGDEST}/lib/bcprov-jdk15on-${BOUNCYCASTLE_VER}.jar"
+		wget -nv -c "$BOUNCYCASTLE_SSL_URL" -O "${SYNOPKG_PKGDEST}/lib/bcpkix-jdk15on-${BOUNCYCASTLE_VER}.jar"
+
+		# Correct the file ownership
```

Comment:
You're already doing this outside of the if-statement, which is a good thing. No need to do it twice.